packages/opc-footer/README.md

```diff
@@ -1,112 +1,138 @@
-# This is opc-footer 👋
+# opc-footer component 👋
 
-![Version](https://img.shields.io/badge/version-0.0.1-blue.svg?cacheSeconds=2592000)
+![Version](https://img.shields.io/badge/version-0.0.2-blue.svg?cacheSeconds=2592000)
 ![Build Status](https://travis-ci.org/dwyl/esta.svg?branch=master)
 [![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/op-components/graphs/commit-activity)
 
-opc-footer is a web component based on lit-element. By default it has the light theme and text for copyright.
+A standardized web component based on Lit Element for Red Hat One Platform Footer.
 
-## Install
+# Usage
+### Footer with links:
+* The links needs to have following data type which is defined in `./types.d.ts` file. Below is the mock data example.
+* use the `opcLinkCatagories` setter function to add links.
+```js
+const links = [
+  { "group":"Group name 1",
```

Comment:
I just realized that you're calling this attribute as link-categories then it should be `category` instead of `group`.
As I mentioned in my previous review, can you please make the datatype as below:
```links = [{
"category":"",
"links": [{
  "text": "",
  "href": ""
   }]
}]```